scripts/run

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
Should this file go into Simplified core? It seems generally useful. Content and metadata scripts need this as well.